src/modules/launcher/Plugins/Microsoft.PowerToys.Run.Plugin.WindowsSettings/Helper/TranslationHelper.cs

```diff
@@ -28,62 +28,83 @@ internal static void TranslateAllSettings(in IEnumerable<WindowsSetting>? settin
-                var type = Resources.ResourceManager.GetString(settings.Type);
-
-                if (string.IsNullOrEmpty(area))
-                {
-                    Log.Warn($"Resource string for [Area{settings.Area}] not found", typeof(Main));
-                }
-
                 if (string.IsNullOrEmpty(name))
                 {
-                    Log.Warn($"Resource string for [{settings.Name}] not found", typeof(Main));
+                    Log.Warn($"Resource string for [{settings.Name}] not found", typeof(TranslationHelper));
                 }
 
+                settings.Name = name ?? settings.Name ?? string.Empty;
+
+                // Translate Type (App)
+                var type = Resources.ResourceManager.GetString(settings.Type);
                 if (string.IsNullOrEmpty(type))
                 {
-                    Log.Warn($"Resource string for [{settings.Name}] not found", typeof(Main));
+                    Log.Warn($"Resource string for [{settings.Type}] not found", typeof(TranslationHelper));
                 }
 
-                settings.Area = area ?? settings.Area ?? string.Empty;
-                settings.Name = name ?? settings.Name ?? string.Empty;
                 settings.Type = type ?? settings.Type ?? string.Empty;
 
-                if (!string.IsNullOrEmpty(settings.Note))
+                // Translate Areas
+                if (!(settings.Areas is null) && settings.Areas.Any())
                 {
-                    var note = Resources.ResourceManager.GetString(settings.Note);
-                    if (string.IsNullOrEmpty(note))
+                    var translatedAreas = new List<string>();
+
+                    foreach (var area in settings.Areas)
                     {
-                        Log.Warn($"Resource string for [{settings.Note}] not found", typeof(Main));
+                        if (string.IsNullOrEmpty(area))
+                        {
+                            continue;
+                        }
+
+                        var translatedArea = Resources.ResourceManager.GetString(area);
+                        if (string.IsNullOrEmpty(translatedArea))
+                        {
+                            Log.Warn($"Resource string for [{area}] not found", typeof(TranslationHelper));
+                        }
+
+                        translatedAreas.Add(translatedArea ?? area);
                     }
 
-                    settings.Note = note ?? settings.Note ?? string.Empty;
+                    settings.Areas = translatedAreas;
                 }
 
+                // Translate Alternative names
                 if (!(settings.AltNames is null) && settings.AltNames.Any())
                 {
                     var translatedAltNames = new Collection<string>();
 
                     foreach (var altName in settings.AltNames)
                     {
-                        if (string.IsNullOrWhiteSpace(altName))
+                        if (string.IsNullOrEmpty(altName))
```

Comment:
Why don't we keep `IsNullOrWhiteSpace` here?

Reply:
Because it will be filled in `TranslationHelper` with `string.Empty` if it's not existing in the json. So I saw ne reason why this can be a white space. And for me a white space is not the same as empty string. Additionally I want to align the code.

Reply:
What do you think makes here more sense? Check strings against NullOrEmpty or against NullOrWhitespace. Some of the values (Name, Type) are initialized with empty string by default.

But I am not sure what happened on white space values in json. 🤔

Reply:
I would say let's keep `IsNullOrWhiteSpace` if there are no particular reasons not to.